src/main/java/tornadofx/Builders.kt

```diff
@@ -109,6 +109,12 @@ fun Pane.progressbar(property: Property<Double>, op: (ProgressBar.() -> Unit)? =
     progressProperty().bind(property)
 }
 
+fun Pane.slider(min: Double? = null, max: Double? = null, value: Double? = null, op: (Slider.() -> Unit)? = null) = opcr(this, Slider().apply {
```

Comment:
Or should I split it into two constructor (like JavaFX) with no support for null parameters?  It would slightly optimize the calls to `adjustValues()`